packages/connect-react/src/components/login/LoginInitScreen.tsx

```diff
@@ -36,12 +36,22 @@ const LoginInitScreen = () => {
         log.debug('fallback: login not allowed');
         navigateToScreen(LoginScreenType.Invisible);
         config.onFallback('');
+        config.onLoaded('loaded successfully', true);
         return;
       }
 
-      if (flags.hasSupportForConditionalUI()) {
+      const lastLogin = getConnectService().getLastLogin();
+
+      if (lastLogin) {
+        navigateToScreen(LoginScreenType.Passkey);
+
+        config.onLoaded('loaded successfully', false);
+        return;
+      } else if (flags.hasSupportForConditionalUI()) {
         log.debug('starting conditional UI');
         void startConditionalUI(res.val.conditionalUIChallenge);
+
+        config.onLoaded('loaded successfully', false);
```

Comment:
We should trigger onLoaded also when conditionalUI is not available (conditional UI is an optional feature, not all users will have it but they can still use passkeys).